plugins/gtav/gtav.cpp

```diff
@@ -3,118 +3,208 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-#define MUMBLE_ALLOW_DEPRECATED_LEGACY_PLUGIN_API
-#include "mumble_legacy_plugin.h"
+#include "MumblePlugin_v_1_0_x.h"
 
-#include "mumble_positional_audio_main.h"  // Include standard positional audio header.
-#include "mumble_positional_audio_utils.h" // Include positional audio header for special functions, like "escape".
+#include "ProcessWindows.h"
+#include "mumble_positional_audio_utils.h"
 
-#include <algorithm> // Include algorithm header for the game version detector
+#include <cstring>
 
-static int fetch(float *avatar_pos, float *avatar_front, float *avatar_top, float *camera_pos, float *camera_front,
-				 float *camera_top, std::string &context, std::wstring &identity) {
-	for (int i = 0; i < 3; i++) {
-		avatar_pos[i] = avatar_front[i] = avatar_top[i] = camera_pos[i] = camera_front[i] = camera_top[i] = 0.0f;
+static std::unique_ptr< ProcessWindows > proc;
+static procptr_t moduleBase = 0;
```

Comment:
```suggestion
static procptr_t moduleBase = nullptr;
```

Reply:
It's not a pointer.

Reply:
Lol then the naming seems to be very misleading. `ptr` usually indicates pointer-type :eyes:

Reply:
In this case `ptr` means that the variable can be used to store a memory address.